docs/testrunner-toolkit/saucectl.md

```diff
@@ -113,7 +106,7 @@ saucectl run --suite <suite_name>
 
 ### `test-env`
 
-<p><small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight></small></p>
+<p xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"><small>supported frameworks: <span class="highlight cypress">cypress</span></small></p>
```

Comment:
What is this reference inside the paragraph tag? Why is it here twice?

Reply:
I have no idea what this is: `xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html"`, we can remove it.